src/main/java/seedu/duke/controllers/DishController.java

```diff
@@ -0,0 +1,102 @@
+package seedu.duke.controllers;
+
+import seedu.duke.entities.Dish;
+import seedu.duke.manager.DishManager;
+
+import java.util.Scanner;
+
+/**
+ * DishController (or MenuController if you like).
+ */
+public class DishController extends Controller {
+    private static final String[] defaultChoice = {"Exit Menu", "Add Dish", "Delete Dish",
```

Comment:
probably change this to `CHOICES` since it is final.

Reply:
the name i mean

Reply:
> the name i mean

Ahh thx